### DISCUSSION TITLE
Error when installing with Amazon S3 as storage driver

### DISCUSSION THREAD

**Discussion by Kieran-C** [SergiX44/XBackBone#385] [Q&A]
I am trying to install XBackBone using S3 as the storage driver. I have it running in a docker container fine, however, when I fill in all the details for my AWS S3 bucket I get the following error:

`Storage setup error: File not found at path: storage_test.txt [0]`

What would be causing this, advice appreciated

Thanks

**Comment by sergix44** [comment]
There is probably an configuration issue on the bucket, it is not able to write/read file from the storage
